backend/kernelCI_app/helpers/errorHandling.py

```diff
@@ -1,4 +1,9 @@
 from django.http import JsonResponse
+from enum import Enum
+
+
+class StatusCodes(Enum):
+    BAD_REQUEST = 400
```

Comment:
nit: is there a reason to create your enum instead of using `http.HTTPStatus` (https://docs.python.org/pt-br/3/library/http.html#http.HTTPStatus) enum?